DexterHuang/CyberCodeOnline
# Improved dungeon validation + removed duplicates

Implemented complete dungeon validation to validate all merge request

Validations:

- duplicate check
- check door position
- check border
- check connection between doors
- check that all free cells are connected

Removed duplicates which were found with the duplicate check
